docs/source/tutorials.data.rst

```diff
@@ -0,0 +1,42 @@
+Manipulating audio tensors
```

Comment:
can we keep the names for the sections more consistent (all nouns) and rename this to something like Audio Tensor Manipulations?